packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -51,10 +51,10 @@ export default class JournalStore {
 
         let error_message = data;
         let classname = '';
-        if (typeof data !== 'string') {
+        if (typeof data === 'object') {
             const { error, message } = data;
             ({ className: classname } = data);
-            error_message = error && error.error ? error.error.message : message;
+            error_message = error && error.error ? error.error.message : message.toString();
```

Comment:
This will kill the component idea we were talking about yesterday, better to add conditions for each type, and then a default one which just pushes the `message` as is.

```JavaScript
if (typeof data === 'boolean') {
    // Return stringified version of message.
} else if (typeof data !== 'string') {
    // Return message as is.
} // etc.
```

Reply:
ahh, you're right. Do you think we should just combine this to journal restructure card as well? It's gonna need refactor anyway.

Reply:
Good idea 👍